service/version.go

```diff
@@ -252,6 +252,12 @@ func (uis *UIServer) modifyVersion(w http.ResponseWriter, r *http.Request) {
 				http.Error(w, err.Error(), http.StatusInternalServerError)
 				return
 			}
+			if projCtx.ProjectRef.CommitQueue.Enabled {
```

Comment:
if I'm reading this correctly, this will only happen if the user elected to abort running tasks, whereas the problem would have happened even if the version is deactivated without aborting running tasks.

Reply:
My read of what happened was that unscheduling the version deactivates the builds
https://github.com/evergreen-ci/evergreen/blob/612c4407f3065a13c95d1f1a1d79906782075564/model/lifecycle.go#L73
so when we check if the version's builds are finished
https://github.com/evergreen-ci/evergreen/blob/612c4407f3065a13c95d1f1a1d79906782075564/model/build/build.go#L85
we see they're not.